README.md

```diff
@@ -1,2 +1,64 @@
 # ibc-apps
-IBC applications and middleware for Cosmos SDK chains
+
+![IBC-APPS Header](ibc-apps.png)
+
+IBC applications and middleware for Cosmos SDK blockchains
+
+🌌 Why have an ibc-apps repo?
```

Comment:
This paragraph and the next talk about IBC and ibc-go without first defining them, how about we swap the order?

1. What is IBC? (short and succinct)
2. Why have an ibc-apps repo?
3. Who's it for?
4. How to use this repo?
5. Bonus content